mmseg/apis/train.py

```diff
@@ -70,7 +71,14 @@ def train_segmentor(model,
     # build runner
     optimizer = build_optimizer(model, cfg.optimizer)
 
-    runner = IterBasedRunner(
+    if runner_type == 'iter':
```

Comment:
By default, all runners are `IterBaseRunner`. 
We may use `IterBaseRunner` if no `runner_type` is not specified.